mmv1/products/gkehub2/Feature.yaml

```diff
@@ -195,6 +200,59 @@ properties:
                       - :MODE_UNSPECIFIED
                       - :COPY
                       - :MOVE
+      - !ruby/object:Api::Type::NestedObject
+        name: clusterupgrade
+        description: Clusterupgrade feature spec.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'upstreamFleets'
+            description: |
+              Specified if other fleet should be considered as a source of upgrades. Currently, at most one upstream fleet is allowed. The fleet name should be either fleet project number or id.
+            required: true
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::NestedObject
+            name: 'postConditions'
+            description: |
+              Post conditions to override for the specified upgrade.
+            required: true
+            default_from_api: true
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'soaking'
+                description: |
+                  Amount of time to "soak" after a rollout has been finished before marking it COMPLETE. Cannot exceed 30 days.
+                required: true
+          - !ruby/object:Api::Type::Array
+            name: 'gkeUpgradeOverrides'
+            description: |
+              Configuration overrides for individual upgrades.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'upgrade'
+                  description: |
+                    Which upgrade to override.
+                  required: true
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'name'
+                      description: |
+                        Name of the upgrade, e.g., "k8s_control_plane". It should be a valid upgrade name. It must not exceet 99 characters.
+                    - !ruby/object:Api::Type::String
+                      name: 'version'
+                      description: |
+                         Version of the upgrade, e.g., "1.22.1-gke.100". It should be a valid version. It must not exceet 99 characters.
```

Comment:
Can we make at least one of these subfields required to prevent users from sending empty blocks?

Reply:
Done, made both of the fields required